src/main/java/seedu/address/ui/TaskViewPanel.java

```diff
@@ -137,6 +137,8 @@ private void handleTaskPanelSelectionChangedEvent(TaskPanelSelectionChangedEvent
     }
     @Subscribe
     public void handleTaskManagerChangedEvent(TaskManagerChangedEvent tmce) {
+        //Try catch clause used here as both complete and dependency has several intermediate states of the task
+        //manager that might trigger this event and throw and exception.
```

Comment:
`throw an exception` not `and`.

Also this comment confuses me, what do you mean?

Reply:
Ill explain face to face? 

Latest commit resolved spelling